packages/passwordless-auth-construct/src/types.ts

```diff
@@ -123,6 +123,32 @@ export type MagicLinkAuthOptions = {
   fromAddress: string;
 };
 
+/**
+ * Options for email user verification
+ */
+export type EmailUserVerificationOptions = {
+  emailSubject?: string;
+  emailBody?: string;
+};
+
+/**
+ * Options for SMS user verification
+ */
+export type SmsUserVerificationOptions = {
+  smsMessage?: string;
+};
+
+/**
+ * Options for OTP user verification
+ */
+export type OtpUserVerification = EmailUserVerificationOptions &
```

Comment:
nit: Don't think `UserVerification` fits well here. Maybe `MessageOptions`? Doesn't need to be changed now, I think these will get ironed out in API review.